examples/cli/discovery.md

```diff
@@ -192,7 +237,7 @@ You can leverage the model hashes feature of `turnkey` to filter which models ar
 If you wanted to only build and benchmark `another_pytorch_model`, you could use this command, which filters `two_models.py` with the hash `215ca1e3`:
 
 ```
-turnkey benchmark scripts/two_models.py::215ca1e3
+turnkey benchmark scripts/two_models.py::c69b7dea
```

Comment:
The prior hash got stale at some point and needed to be replaced.

At some point we might want to auto-generate the docs to ensure correctness of printouts and variables ;)